devops-engineer.md

```diff
@@ -0,0 +1,129 @@
+We're looking for a
+
+# Talented DevOps Engineer
+
+## About us
+
+**CLASTIX** is a young Italian technology startup that helps enterprises to
+adopt _Cloud Native_ solutions through an innovative set of products and
+services.
+
+**CLASTIX** develops and integrates open source software projects for automated
+operations across cloud, edge, and data-centers. We support customers of any
+size to adopt modern infrastructures and embrace the Cloud Native revolution.
+
+We believe in continuous learning, flexibility, agile methodology, sharing
+ideas and knowledge, supporting each other, and pushing ourselves to the next
+level.
+ 
+## What we are looking for
+
+We are creating a cutting-edge, end-to-end innovative Cloud Native ecosystem
+on top of Kubernetes and other open-source projects and to accomplish this,
+we're looking for a talented DevOps Engineer. If you share our passions and
+want to make part of this journey, please consider joining us.
+ 
+
+## What you will do
+
+You will be a key part of a talented and successful engineering team that
+works together with important customers to build cutting-edge technology
+solutions based on our open source stack.
+
+You are required to:
+ 
+- Design, build, and operate fully automated, highly scalable, efficient,
+  and reliable customer infrastructures based on Kubernetes and Cloud Services.
+- Design, build and operate infrastructures as a code across cloud, edge,
+  and data-centers. 
+- Work with our Customers and bring a DevOps perspective into their teams
+  (~70% customer-facing).
+- Collect suggestions and improvements coming from the Customer's feedback and
+  requirements and thus help us to provide new products and better solutions.
+- Being tech-savvy and a lifelong learner, stay updated on new technologies and
+  trends.
+ 
+## Who you are
+
+We would like someone who is excited by the opportunity to work in a rapidly
+evolving startup. A perfect candidate needs to master most of the following
+subjects:
+
+- DevOps culture, principles, and tools
+- Container runtimes (_CRI-O_, _containerd_, _Docker_)
+- _Kubernetes_ ecosystem (_vanilla_, _OCP_, _Rancher_) and related interfaces
+  (_CSI_, _CNI_, _SMI_)
+- Unix-like OS and shell scripting (_bash_, _shell_, _zsh_)
+- Configuration Management (_Ansible_, _Puppet_, _Chef_)
+- Major Public Cloud services (_AWS_, _GCP_, _Azure_)
+- Virtualization (_VMware_, _KVM_, _RedHat Enterprise Virtualization_)
+- Software-Defined Computing, Networking, and Storage solutions (_OpenStack_,
+  _BGP_, _VXLAN_)
```

Comment:
Software-Defined Computing, Networking, and Storage solutions (_Open Virtual Switch_, _VXLAN_, _BGP_, _Ceph_, _Object Storage_)